<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1734</width>
    <height>1054</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Mono</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>compareNavPlot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Data Workspace</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="imuFile">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>IMU File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ResultFile">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Result File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="RefFile">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Refenrece File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="StateFile">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Nav State File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QToolButton" name="c1">
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="c2">
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="c3">
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="c4">
           <property name="maximumSize">
            <size>
             <width>36</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLineEdit" name="imuFileHeaderline">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="placeholderText">
            <string>h</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ResultFileHeaderline">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>2</string>
           </property>
           <property name="placeholderText">
            <string>h</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="RefFileHeaderline">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>2</string>
           </property>
           <property name="placeholderText">
            <string>h</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="StateFileHeaderline">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>2</string>
           </property>
           <property name="placeholderText">
            <string>h</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QComboBox" name="gUnit">
           <item>
            <property name="text">
             <string>deg/s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>rad/s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>deg</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>rad</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QLineEdit" name="RefDataCol">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1,2,3,4,5,6,7,8,9,10,17,18,19,20</string>
           </property>
           <property name="placeholderText">
            <string>t,px,py,pz,vx,vy,vz,p,r,y,mea,nsat,pdop,amb</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="imuDataCol">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1,2,3,4,5,6,7</string>
           </property>
           <property name="placeholderText">
            <string>t,gx,gy,gz,ax,ay,az</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="imuSampleRate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>151</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>100</string>
           </property>
           <property name="placeholderText">
            <string>Sample Rate</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="aUnit">
           <item>
            <property name="text">
             <string>m/s2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>m/s</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QLineEdit" name="ResultDataCol">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1,2,3,4,5,6,7,8,9,10,17,18,19,20</string>
           </property>
           <property name="placeholderText">
            <string>t,px,py,pz,vx,vy,vz,p,r,y,mea,nsat,pdop,amb</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QComboBox" name="RefAmb">
           <item>
            <property name="text">
             <string>Fixed</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="StateDataCol">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1,18,19,20</string>
           </property>
           <property name="placeholderText">
            <string>t,nsat,pdop,amb</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="ResultAmb">
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fixed</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Figure Workspace</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="3">
          <widget class="QGraphicsView" name="fig_h">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QGraphicsView" name="fig_dp">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGraphicsView" name="fig_state">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGraphicsView" name="fig_g">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QGraphicsView" name="fig_da">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QGraphicsView" name="fig_dv">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGraphicsView" name="fig_bl">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGraphicsView" name="fig_a">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1734</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProc">
    <property name="title">
     <string>Proc</string>
    </property>
    <addaction name="actionImport_File"/>
    <addaction name="actionPlot"/>
    <addaction name="actionClear_Plot"/>
   </widget>
   <widget class="QMenu" name="menuOutput">
    <property name="title">
     <string>Output</string>
    </property>
    <addaction name="actionSavfig"/>
   </widget>
   <addaction name="menuProc"/>
   <addaction name="menuOutput"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actioniM">
   <property name="text">
    <string>iM</string>
   </property>
  </action>
  <action name="actionImport_File">
   <property name="text">
    <string>Import File</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="text">
    <string>Plot</string>
   </property>
  </action>
  <action name="actionSavfig">
   <property name="text">
    <string>Savfig</string>
   </property>
  </action>
  <action name="actionClear_Plot">
   <property name="text">
    <string>Clear Plot</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
